manifests/base/README.md

```diff
@@ -51,7 +51,7 @@ patchesJson6902:
         value: "certauth.<YOUR_DOMAIN>"
 ```
 
-NOTE: If you decide to use something like [ksops][ksops] for your secret management or want to use a generator, you need
+NOTE: If you decide to use something like [ksops](https://github.com/viaduct-ai/kustomize-sops) for your secret management or want to use a generator, you need
```

Comment:
Can you change this to use the reference instead? That was the original intention, but clearly I missed it.